HelpSource/Overviews/Operators.schelp

```diff
@@ -303,7 +303,8 @@ method:: /
 Division.
 
 method:: %
-Floating point modulo.
+Modulo.
+NOTE:: Uses floored division. Integer modulo used to exhibit unexpected behavior for negative modulus; see link::Classes/Integer#-mod_old::.::
```

Comment:
How about we explicitly add the version name here?

Suggested:
`Integer modulo used to exhibit unexpected behavior for negative modulus before SC 3.14; see link::Classes/Integer#-mod_old::.::